starknet_py/tests/e2e/test-variables.env.template

```diff
@@ -0,0 +1,6 @@
+INTEGRATION_NODE_URL=URL
+TESTNET_NODE_URL=URL
+INTEGRATION_ACCOUNT_PRIVATE_KEY=0x1234567890
+INTEGRATION_ACCOUNT_ADDRESS=0x9876543210
+TESTNET_ACCOUNT_PRIVATE_KEY=0xabcdef
+TESTNET_ACCOUNT_ADDRESS=0xfedcba
```

Comment:
So running the tests will fail because these are not set?
Maybe it would be better to not make this a template, but an actual file with some dummy values?